code_snippets/Deploy_Previously_Created_Model.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>Can you pull the DEMO-semantic-segmentation-config- out and generalize?</p><p>Seems like you can have a config section at the top and be clear about user-supplied values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>